modules/runner/src/main/java/org/apache/ignite/internal/app/IgnitionImpl.java

```diff
@@ -80,11 +80,14 @@
     private static final String VER_KEY = "version";
 
     /** {@inheritDoc} */
-    @Override public synchronized Ignite start(String jsonStrBootstrapCfg) {
+    @Override public synchronized Ignite start(String nodeName, String jsonStrBootstrapCfg) {
+        assert !StringUtil.isNullOrEmpty(nodeName) : "Node local name is empty";
```

Comment:
I would add @NotNull for nodeName here and in ancestor, and also reflect that facts in javadocs